documents/operating.md

```diff
@@ -231,6 +231,9 @@ Projects that are funded by third parties through grants or as a product for hir
 
 Corporate Overlords may apply for grants and seek out contracts with third parties so long as these efforts have been approved by the Overmind.
 
+### The BIFFUD Operating Branch
+The BIFFUD Operating Branch is the collective set of processes (and people) associated with corporate operations.  This includes things like invoicing, contracting, purachases, expenses, and filing taxes.  The nature and explicit responsibilities of the BIFFUD Operating Branch is determined by the Overmind with guidance from the Hive Mind.
```

Comment:
Just defining the concept of the operating branch (it isn't creating something new, just codifying the way we're operating now)